test/e2e/testdata/simple-fallback.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: numaflow.numaproj.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: simple-fallback
+spec:
+  vertices:
+    - name: in
+      source:
+        http: {}
+    - name: cat
+      scale:
+        min: 1
+      udf:
+        builtin:
+          name: cat
+    - name: output
+      scale:
+        min: 1
+      sink:
+        udsink:
+          container:
+            image: quay.io/numaio/numaflow-go/fb-sink-log:stable
```

Comment:
nit: could you please add the source code to numaflow-go? seems you are manually building it by uncomment the simple sink. Might be better if we have a separate example so that we utilize all the benefits of CI improvements.

Reply:
https://github.com/numaproj/numaflow-go/pull/122